api/src/HttpCache/ResponseTagger.php

```diff
@@ -35,12 +34,6 @@ public function addTags(array $tags) {
     private function isCacheable(): bool {
         $request = $this->requestStack->getCurrentRequest();
 
-        if (!$request->isMethodCacheable()) {
-            return false;
-        }
-
-        $requestUri = $request->getRequestUri();
-
-        return (bool) preg_match('{'.$this->matchPath.'}', $requestUri);
```

Comment:
Now we're adding the tags for all GET requests. Was this done by purpose? We have some get requests with a lot of items which caused the header to get too large. We should carefully verify this, before enabling tag responses for all requests.

Reply:
We generated them anyway, just did not add them.
Since
- #5910
we can disable sending them to the nginx, which crashed on staging.
This was for now on purpose, because it also used the regex, which does not scale in my opinion.

Reply:
It's still possible that varnish could crash when the header is too large. I don't recall the exact limit, but I have definitely experienced this during initial setup of varnish.